subsys/bluetooth/mesh/shell.c

```diff
@@ -1341,10 +1452,15 @@ static int cmd_app_key_get(const struct shell *shell, size_t argc, char *argv[])
 	uint16_t keys[16];
 	size_t cnt;
 	uint8_t status;
-	int err, i;
+	int err = 0, i;
```

Comment:
Here and below. Split into 2 lines to avoid declaring initialized and non-initialized variables on the same line.